hms-app/src/actions/HackathonActions.tsx

```diff
@@ -1,8 +1,13 @@
-import { Hackathon, parseHackathon, parseHackathons } from '../common/types'
+import {
+  Hackathon,
+  HackathonPreview,
+  parseHackathon,
+  parseHackathons,
+} from '../common/types'
 
 const core_url = process.env.REACT_APP_CORE_URL
 
-export const getListOfHackathons = (): Promise<Hackathon[]> => {
+export const getListOfHackathons = (): Promise<HackathonPreview[]> => {
```

Comment:
but `parseHackathons` returns a `Hackathon[]`?  I would think we need another function

Reply:
I think parseHackathons then should probably return HackathonPreview? Every list only return previews right?

Reply:
yes it should have the return type changed, but then the name too.  That way it is obvious it returns `HackathonPreview[]` and not `Hackathon[]`